script/upgrades/GHSWhitelist/GHSWhitelist.sol

```diff
@@ -0,0 +1,181 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// solhint-disable func-name-mixedcase, contract-name-camelcase, function-max-lines, var-name-mixedcase
+pragma solidity ^0.5.13;
+pragma experimental ABIEncoderV2;
+
+import { GovernanceScript } from "script/utils/Script.sol";
+import { console } from "forge-std/console.sol";
+import { Contracts } from "script/utils/Contracts.sol";
+import { Chain } from "script/utils/Chain.sol";
+import { Arrays } from "script/utils/Arrays.sol";
+
+import { IChainlinkRelayerFactory } from "mento-core-2.5.0/interfaces/IChainlinkRelayerFactory.sol";
+import { IChainlinkRelayer } from "mento-core-2.5.0/interfaces/IChainlinkRelayer.sol";
+
+import { IMentoUpgrade, ICeloGovernance } from "script/interfaces/IMentoUpgrade.sol";
+
+interface ISortedOracles {
+  function addOracle(address, address) external;
+
+  function removeOracle(address, address, uint256) external;
+
+  function setEquivalentToken(address, address) external;
+
+  function getEquivalentToken(address) external returns (address);
+
+  function medianRate(address) external returns (uint256, uint256);
+
+  function getOracles(address) external returns (address[] memory);
+
+  function setTokenReportExpiry(address, uint256) external;
+
+  function getTokenReportExpirySeconds(address) external returns (uint256);
+
+  function tokenReportExpirySeconds(address) external returns (uint256);
+}
+
+/**
+ forge script {file} --rpc-url $ALFAJORES_RPC_URL 
+                     --broadcast --legacy 
+ * @dev depends on: ../deploy/*.sol
+ * @dev Script to whitelist the GHS relayers via Celo Governance
+ */
+contract GHSWhitelist is IMentoUpgrade, GovernanceScript {
+  using Contracts for Contracts.Cache;
+
+  bool public hasChecks = true;
+  ICeloGovernance.Transaction[] private transactions;
+
+  // Mento contracts
+  IChainlinkRelayerFactory private relayerFactory;
+  ISortedOracles private sortedOracles;
+  address private cGHS;
+
+  mapping(address => IChainlinkRelayer) private relayersByRateFeedId;
+
+  function prepare() public {
+    loadDeployedContracts();
+    setAddresses();
+  }
+
+  /**
+   * @dev Loads the deployed contracts from previous deployments
+   */
+  function loadDeployedContracts() public {
+    contracts.loadSilent("MU07-Deploy-ChainlinkRelayerFactory", "latest");
+    contracts.loadSilent("cGHS-00-Deploy-Proxy", "latest");
+  }
+
+  /**
+   * @dev Sets the addresses of the various contracts needed for the proposal.
+   */
+  function setAddresses() public {
+    relayerFactory = IChainlinkRelayerFactory(contracts.deployed("ChainlinkRelayerFactoryProxy"));
+    sortedOracles = ISortedOracles(contracts.celoRegistry("SortedOracles"));
+    cGHS = contracts.deployed("StableTokenGHSProxy");
+
+    // TODO: Maybe confirm this is not the pilot address
```

Comment:
you still wanna do this or can this TODO go?

Reply:
Can go